public/js/main.js

```diff
@@ -157,19 +172,22 @@ async function forkCards (forkList) {
   })
 }
 
+// Gets .manifest.json from a given fork
 async function getManifest (forkFullName, branch = 'master') {
   return await fetchJSON(`https://raw.githubusercontent.com/${forkFullName}/${branch}/.manifest.json`)
 }
 
+// Fetches a codeSnippet for a given fork
 async function getCodeSnippet (forkFullName, branch = 'master') {
   const manifest = await getManifest(forkFullName)
+  // Gets the raw file content from a path provided in .manifest.json
   const codeSnippetPromise = await fetch(`https://raw.githubusercontent.com/${forkFullName}/${branch}/${manifest.filePath}`)
   const codeSnippet = await codeSnippetPromise.text()
```

Comment:
```suggestion
  const codeSnippet = await codeSnippetPromise.text()
  codeSnippet.trim()
```

We should still trim the snippet if there are whitespace at the end of the file to avoid flooding the page.